### PR TITLE
Bta 665: Implement new VAT subpage design

### PR DESCRIPTION
**Pull request by AndrewShanks**

### CODE REVIEW

File: app/views/partials/account_summary/vat/account_summary.scala.html [hunk 1]
```diff
@@ -23,74 +23,75 @@
...
     data-journey-click="vat:Click:Submit your VAT return"
     data-sso="false">@Html(messages("vat.return.complete.link.text")) @partials.open_in_portal_window()
-    </a>
+  </a>
 }
 
 
 @vatVarContent
+<section>
+  <h2 style="margin-top: 1.875em !important">@messages("vat.your_returns")</h2>
```
Comment (gianick) [inline app/views/partials/account_summary/vat/account_summary.scala.html]: Remove inline CSS styles.

File: app/views/partials/account_summary/vat/account_summary.scala.html [hunk 2]
```diff
@@ -23,74 +23,75 @@
...
+    @openPeriods.map { op =>
+        <p>@Html(messages("vat.return.for.period.ready.to.complete", formatLocalDate(op.openPeriod), "<span class=\"flag flag--soon\">", "</span>"))</p>
```
Comment (gianick) [inline app/views/partials/account_summary/vat/account_summary.scala.html]: Is it possible to remove the markup from the message?

File: app/views/partials/account_summary/vat/account_summary.scala.html [hunk 3]
```diff
@@ -23,74 +23,75 @@
...
+  <h2 style="margin-top: 1.875em !important">@messages("vat.your_payments")</h2>
+    <p>
+      <span id="vat-balance">
+      @Html(balanceInformation)
+      </span>
+    </p>
+    <p>
+         @breakdownLink.map { link =>
+      <a id="vat-see-breakdown-link" href="@link"
+        data-journey-click="link - click:VATaccountSummary:how we worked this out/view statement"
```
Comment (gianick) [inline app/views/partials/account_summary/vat/account_summary.scala.html]: Is this a valid GA tracking code?

File: app/views/partials/account_summary/vat/account_summary.scala.html [hunk 4]
```diff
@@ -23,74 +23,75 @@
...
+  @if(showRepaymentContent) {
+      <details id="vat-repaid-details">
+          <summary class="summary--repaid underline">
+              <span id="vat-when-repaid">@Messages("vat.when.repaid")</span>
+          </summary>
+          <div class="panel-indent">
+              <p>
+                @messages("vat.we_will_transfer_this_amount_")
+                <a id="vat-repayments-account"
+                  data-journey-click="link - click:VATaccountSummary:repayments bank account"
```
Comment (gianick) [inline app/views/partials/account_summary/vat/account_summary.scala.html]: Is this a valid GA tracking code?

File: app/views/partials/account_summary/vat/account_summary.scala.html [hunk 5]
```diff
@@ -23,74 +23,75 @@
...
+                  href=@appConfig.getBusinessAccountUrl("manageAccount")#bank>@messages("vat.repayments_bank_account")</a>
+                @messages("vat.if_you_have_set_one_up_")
+              </p>
+              <p>@messages("vat.when.repaid.detail.2")</p>
+              <p>
+                @messages("vat.do_not_get_in_touch_unless_")
+                <a id="vat-more-than-21-days"
+                  target="_blank" rel="external noopener"
+                  data-journey-click="link - click:VATaccountSummary:more than 21 days"
```
Comment (gianick) [inline app/views/partials/account_summary/vat/account_summary.scala.html]: Is the tracking code valid?

File: app/views/partials/account_summary/vat/account_summary.scala.html [hunk 6]
```diff
@@ -23,74 +23,75 @@
...
+                  href=@appConfig.getGovUrl("vatEnquiries")>@messages("vat.more_than_21_days")</a>
+              </p>
+          </div>
+      </details>
+  }
+  <a id="vat-make-payment-link"
+    href="@appConfig.getUrl("makeAPayment")"
+    class="button button--payment"
+    data-journey-click="link - click:VATaccountSummary:Make a VAT payment">
```
Comment (gianick) [inline app/views/partials/account_summary/vat/account_summary.scala.html]: Is this a valid GA tracking code?

File: app/views/partials/account_summary/vat/direct_debit_details.scala.html [hunk 7]
```diff
@@ -17,16 +17,16 @@
...
-<details>
+<details class="subsection">
     <summary class="underline">
         <span id="vat-direct-debit-see-detail">@messages("vat.direct.debit.see.detail")</span>
     </summary>
     <div class="panel-indent">
         <p id="vat-direct-debit-message">@messages("vat.direct.debit.message", details.periodEndDate.toString("dd MMMM yyyy"), details.periodPaymentDate.toString("dd MMMM yyyy"))</p>
         <a id="vat-direct-debit-help-link"
-           data-journey-click="VatDirectDebit:Click:Help"
+           data-journey-click="link - click:VATaccountSummary:Change or cancel your direct debit"
```
Comment (gianick) [inline app/views/partials/account_summary/vat/direct_debit_details.scala.html]: Is this a valid GA tracking code?

File: app/views/partials/account_summary/vat/generic_error.scala.html [hunk 8]
```diff
@@ -18,5 +18,8 @@
 
 @(portalUrl: String)(implicit request: Request[_], messages: Messages)
 <p>@messages("account.summary.genericerror")</p>
-<p>@Html(messages("account.summary.genericerror.portal",
-    "<a id=\"ct-old-hmrc\" href=\"" + portalUrl + "\" data-journey-click=\"CT:click:oldHMRCWebsite\">","</a>"))</p>
+<p>@messages("account_summary.generic_error.try_refreshing_")
+   <a id="vat-old-hmrc" href= @portalUrl data-journey-click="link-click:VATgenericError:old HMRC website"
```
Comment (gianick) [inline app/views/partials/account_summary/vat/generic_error.scala.html]: Is this a valid GA tracking code?

File: app/views/partials/account_summary/vat/prompt_to_activate_direct_debit.scala.html [hunk 9]
```diff
@@ -16,9 +16,10 @@
 
 @import config.FrontendAppConfig
 
-@(appConfig:FrontendAppConfig, vatDecEnrolment: VatDecEnrolment)(implicit messages:Messages, request:Request[_])
+@(appConfig:FrontendAppConfig, vatDecEnrolment: VatDecEnrolment)(implicit request:Request[_], messages:Messages)
 
 <a id="vat-direct-debit-setup-link"
-   data-journey-click="VatDirectDebit:Click:Setup"
-   href='@appConfig.getPortalUrl("vatOnlineAccount")(Some(vatDecEnrolment))'>@messages("vat.direct.debit.setup")
+   data-journey-click="link - click:VATaccountSummary:Set up a Direct Debit"
```
Comment (gianick) [inline app/views/partials/account_summary/vat/prompt_to_activate_direct_debit.scala.html]: Is this a valid GA tracking code?

File: app/views/partials/account_summary/vat/account_summary.scala.html (hunk 4 above)
Comment (AndrewShanks) [reply app/views/partials/account_summary/vat/account_summary.scala.html]: It's my understanding from working with Adam that this is the style we want going forward - I recommend double-checking with him.

File: app/views/partials/account_summary/vat/prompt_to_activate_direct_debit.scala.html [hunk 10]
```diff
@@ -16,9 +16,10 @@
...
+   href='@appConfig.getPortalUrl("vatOnlineAccount")(Some(vatDecEnrolment))'
+   target=\"_blank\" rel=\"external noopener\" > @messages("vat.direct.debit.setup")
```
Comment (gianick) [inline app/views/partials/account_summary/vat/prompt_to_activate_direct_debit.scala.html]: Why are we escaping these?

File: app/views/partials/account_summary/vat/vat_var/vat_var_prompt_to_activate.scala.html [hunk 11]
```diff
@@ -20,10 +20,17 @@
...
+<div class="notice">
+  <span class="icon icon-important">
+    <span class="visuallyhidden">@messages("site.importantinfo")</span>
   </span>
+  <p>
+    <strong>
+      @messages("vat_variations.account_summary.received_an_activation_pin_")
+      <a id="vat-activate-or-enrol-details-summary"
+        data-sso="false" data-journey-click="link - click:VATVar:Enter pin" target="_blank" rel="external noopener"
+        href = @emacUrlBuilder.getActivationUrl("vat-change-details")(Some(vatDecEnrolment))&@appConfig.getReturnUrl(currentUrl) >
```
Comment (gianick) [inline app/views/partials/account_summary/vat/vat_var/vat_var_prompt_to_activate.scala.html]: Please remove spaces

File: app/views/partials/account_summary/vat/account_summary.scala.html (hunk 3 above)
Comment (gianick) [reply app/views/partials/account_summary/vat/account_summary.scala.html]: Can we make sure that `/` is a valid character in a tracking code?

File: app/views/partials/sidebar/filing_calendar_missing.scala.html [hunk 12]
```diff
@@ -19,8 +19,8 @@
 @(appConfig: FrontendAppConfig, vatDecEnrolment: VatDecEnrolment)(implicit request :Request[_], messages:Messages)
 
 <li>
-  <p class="font-xsmall">@messages("subpage.sidebar.interval_unavailable_text") <br>
+  <p>
     @messages("subpage.sidebar.interval_unavailable_text_2") <a id="your-vat-certificate" href="@appConfig.getPortalUrl("vatCertificate")(Some(vatDecEnrolment))"
-    data-sso="false">@messages("subpage.sidebar_certificate_link_text")</a>.
+    data-sso="false", target="_blank" rel="external noopener" data-journey-click="link - click:VATsidebar:VAT certificate" > @messages("subpage.sidebar_certificate_link_text")</a>.
```
Comment (gianick) [inline app/views/partials/sidebar/filing_calendar_missing.scala.html]: Remove unnecessary space `certificate" >`

File: app/views/partials/sidebar_links.scala.html [hunk 13]
```diff
@@ -23,64 +23,35 @@
...
       <h3>@messages("subpage.sidebar.vrn")</h3>
-      <p class="font-xsmall">@vatDecEnrolment.vrn</p>
+      <p>@vatDecEnrolment.vrn</p>
     </div>
 
     <div class="subsection" id="filing-for-vat">
       <h3>@messages("subpage.sidebar.when_you_file_for_vat")</h3>
-        <ul class="sidebar__list">
+        <ul class="list">
             @sidebarSchedule
```
Comment (gianick) [inline app/views/partials/sidebar_links.scala.html]: Switch to 2 spaces indentation please.

File: conf/messages.cy [hunk 14]
```diff
@@ -42,6 +42,7 @@ site.continue = Parhau
 site.service_name = Cyfrif treth busnes
 site.textarea.char_limit = (Limit is {0} characters)
 site.signout = Allgofnodi
+site.importantinfo = Important information
```
Comment (gianick) [inline conf/messages.cy]: Missing Welsh translation

File: conf/messages.cy [hunk 15]
```diff
@@ -53,74 +54,82 @@ links.opensinportal = (yn agor CThEM ar-lein)
...
 
 #######################################################
 ##  Partial
 #######################################################
 
 partial.heading = TAW
-## TODO: get translation for VRN
 partial.vrn = Eich Cyfeirnod Unigryw y Trethdalwr (UTR) yw {0}.
 partial.moredetails = Rhagor o fanylion Treth Gorfforaeth
 partial.changesToVat = Changes to VAT
```
Comment (gianick) [inline conf/messages.cy]: Missing Welsh translation

**Pull request merged**